src/trace-writer.js

```diff
@@ -244,16 +249,19 @@ TraceWriter.prototype.flushBuffer_ = function(projectId) {
  * @param {string} json The stringified json representation of the queued traces.
  */
 TraceWriter.prototype.publish_ = function(projectId, json) {
+  // TODO(ofrobots): assert.ok(this.config_.project), and stop accepting
+  // projectId as an argument.
   var that = this;
   var uri = 'https://cloudtrace.googleapis.com/v1/projects/' +
     projectId + '/traces';
 
-  this.request_({
+  var options = {
     method: 'PATCH',
     uri: uri,
     body: json,
     headers: headers
-  }, function(err, response, body) {
+  };
+  that.request(options, function(err, body, response) {
```

Comment:
I have been scratching my head about how this used to work before, but this does need to be `(err, body, response)`.